plugins/woocommerce/client/legacy/js/frontend/single-product.js

```diff
@@ -213,7 +213,16 @@ jQuery( function( $ ) {
 		// But only zoom if the img is larger than its container.
 		if ( zoomEnabled ) {
 			var zoom_options = $.extend( {
-				touch: false
+				touch: false,
+				callback: function() {
+					var zoomImg = this;
+					
+					setTimeout( function() {
+						zoomImg.removeAttribute( 'role' );
+						zoomImg.setAttribute( 'alt', '' );
+						zoomImg.setAttribute( 'aria-hidden', 'true' );
```

Comment:
`aria-hidden="true"` should hide the image for assistive technologies. However, since it's no widely supported, the removal of the `role="presentation"` attribute and the empty `alt` value ensures the zoomed-in image is interpreted as decorative.

In fact, the `role="presentation"` attribute and `alt` with a valid value cannot coexist since they provide opposite information to assistive technologies.

**Reference**:
https://www.w3.org/WAI/tutorials/images/decorative/